reobin/vimcolorschemes
# cypress setup & loading the app

## Type of PR


## Description
- cypress setup
- a simple test to load up the app  

## Related issue

 this [issue](https://github.com/reobin/vimcolorschemes/issues/254)

## QA Instructions

Since, you mentioned to load up app & test for anything present there, is it good to check for header logo content as
![image](https://user-images.githubusercontent.com/44117648/96311092-5a04d580-1026-11eb-8936-ceb7bd024dca.png)


## Added unit tests?


## Added documentation?
